avm/res/db-for-my-sql/flexible-server/main.bicep

```diff
@@ -407,6 +412,13 @@ module flexibleServer_administrators 'administrator/main.bicep' = [
   }
 ]
 
+module flexibleServer_advancedThreatProtection 'advanced-threat-protection/main.bicep' = if (advancedThreatProtectionEnabled) {
```

Comment:
I wonder if the boolean flag should be passed through to enable users to also disable the feature again (if that's even possible). Thoughts?

Reply:
Sounds good to me. I've updated the module accordingly.